include/glow/Backends/TraceEvents.h

```diff
@@ -56,9 +56,14 @@ struct TraceEvent {
 
   static void dumpTraceEvents(std::vector<TraceEvent> &events,
                               llvm::StringRef filename);
+
+  static uint64_t now();
 };
 
-/// Tracing / Profiling events map for a compiled function.
+/// Tracing / Profiling events map for a CompiledFunction.
+///   This class encodes information on how to read event metrics out of Tensors
```

Comment:
is this indentation intended?

Reply:
Probably not, been awhile since I wrote this. I'll remove it